src/routes/controllers/articles.controller.js

```diff
@@ -7,6 +7,7 @@ import {
   errorResponse,
   verifyToken,
   getArticlebyId,
+  calcAverageRating,
```

Comment:
I think this function can be inside the rating controller if it is not being used or re-used in some place else. Instead of having it in the global helpers. Or what do you think?

Reply:
Fair point. We can have it in a routes helper file. The function however can stand independent of the rating controller, hence its localisation.